lib/trento/router.ex

```diff
@@ -106,4 +106,6 @@ defmodule Trento.Router do
            ],
            to: Databases.Database,
            lifespan: Databases.Lifespan
+
+  middleware OpentelemetryCommanded.Middleware
```

Comment:
Example of what can be seen in the traces:
![image](https://github.com/trento-project/web/assets/18139046/45a4927c-30eb-40cc-a1fa-aa229265609c)